sdk/build.gradle

```diff
@@ -1,20 +1,25 @@
+import io.radar.mvnpublish.MavenServer
+
 apply plugin: "com.android.library"
 apply plugin: "kotlin-android"
 apply plugin: "kotlin-android-extensions"
 apply plugin: "org.jetbrains.dokka"
-apply plugin: "maven-publish"
-apply plugin: "signing"
+apply plugin: 'io.radar.mvnpublish'
+apply plugin: 'io.radar.ghpublish'
 
 ext {
     radarVersion = "3.2.2"
 }
+String branch = System.getenv('CIRCLE_BRANCH') ?: ''
+boolean publishAsDraft = (branch != 'master')
+String buildNumber = publishAsDraft ? ".${System.getenv('CIRCLE_BUILD_NUM') ?: System.currentTimeMillis()}" : ''
```

Comment:
This, paired with the branch options in circleci config file, will support a flow whereby we use a _develop_ branch to create draft builds or releases, and the _master_ branch will perform full releases.

Reply:
given it's an open-source project, it's pretty common for developers to open PRs against a `master` branch. I think it'd be better to use `master` as the draft branch and a separate `production` branch for full releases.

Reply:
either works for me. I updated this config to support that branching model as well.